etl-job/fhir_import_export.py

```diff
@@ -141,6 +188,28 @@ def _load_all(study, project_id, output) -> bool:
     return True
 
 
+def _get(input_data, output, program, project, user) -> str:
+    """Export data from the fhir store to bucket, returns object_id."""
+    can_read = _can_read(output, program, project, user)
+    if not can_read:
+        output['logs'].append(f"No read permissions on {program}-{project}")
+        return None
+
+    study_path = f"studies/{project}"
+    project_id = f"{program}-{project}"
+
+    logs = fhir_get(f"{program}-{project}", study_path, DEFAULT_ELASTIC)
+    output['logs'].extend(logs)
+
+    # zip and upload the exported files to bucket
+    config = Config()
+    cp_result = cp(config=config, from_=study_path, project_id=project_id, ignore_state=False)
```

Comment:
Looks good! Does `ignore_state=False` mean it overwrites existing data or keeps it in place?

https://github.com/ACED-IDP/gen3_util/blob/main/gen3_util/meta/uploader.py#L31

Reply:
over writes existing data.  will improve docs in next PR